news/PR1114.dev

```diff
@@ -0,0 +1,2 @@
+Introduced static-type checking through inclusion of mypy in tox.
```

Comment:
I'm not sure if towncrier can process multiline news file. Let's keep this only with mypy update.

Reply:
Could you try to run `towncrier --draft` to test this?

Reply:
I tried it and it doesn't seems to be an issue.